package.json

```diff
@@ -6,6 +6,7 @@
     "build": "gatsby build",
     "develop": "gatsby develop",
     "dev": "gatsby develop",
+    "ssrdev": "export DEV_SSR=true && gatsby develop",
```

Comment:
I think this could break on windows environments

Reply:
Set it true by default with an option to opt-out by setting an environment variable on .env.